templates/index.html

```diff
@@ -11,7 +11,7 @@
 SYNOPSIS
     curl -F 'name=@/path/to/image' {{ url }}
 
-    Or upload via your browser: <form action="/" method="post" enctype="multipart/form-data" style="display: inline;"><input type="file" name="image" width="10" multiple accept="image/*"/><input type="submit" value="Upload!"/></form>
+    Or upload via your browser: <form action="/" method="post" enctype="multipart/form-data" style="display: inline;"><input type="file" name="image" width="10" multiple accept="image/*"/>{% if password_required %}<input type="password" name="password" style="width: 20px"/>{% end %}<input type="submit" value="Upload!"/></form>
```

Comment:
We can add a `required` attribute so that a random user won't submit invalid data.

Reply:
done